### PR TITLE
[MS-2] Adding support for the horizontal layouts and rotation

### PR DESCRIPTION
**Pull request by alexandr-simprints**
As we are striving to improve the UX within the SID application, it is necessary to start targeting tablet devices alongside regular phones.

One of the key differences from phone usage is that tablets are often used horizontally (in landscape mode). However, currently, the SID provides only portrait (vertical) orientation support, making it inconvenient to use the application sometimes, having to physically rotate the device if the callout application is used in landscape mode.

This epic aims to address this UX concern and approach the implementation by initially adding the landscape support individually for each fragment of the biometrics flow (i.e. `LoginFragment`, `ConsentFragment`, etc.), and then by adding the landscape support for the hosting `OrchestratorActivity`

### CODE REVIEW

File: feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorFragment.kt [hunk 1]
```diff
@@ -77,7 +77,7 @@ internal class OrchestratorFragment : Fragment(R.layout.fragment_orchestrator) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         if (savedInstanceState != null) {
-            orchestratorVm.requestProcessed = savedInstanceState.getBoolean(KEY_REQUEST_PROCESSED)
+            orchestratorVm.isRequestProcessed = savedInstanceState.getBoolean(KEY_REQUEST_PROCESSED)
```
Comment (luhmirin-s) [inline feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorFragment.kt:80]: Is this the only actual change in the file and all other changes are essentially just adding extra line breaks?

File: fingerprint/connect/src/main/java/com/simprints/fingerprint/connect/screens/controller/ConnectScannerControllerFragment.kt [hunk 2]
```diff
@@ -68,15 +71,20 @@ internal class ConnectScannerControllerFragment : Fragment(R.layout.fragment_con
     ) { permissions: Map<String, Boolean> ->
         val permission = permissions
             .filterKeys(bluetoothPermissions::contains)
-            .map { (permission, isGranted) -> requireActivity().permissionFromResult(permission, isGranted) }
+            .map { (permission, isGranted) ->
```
Comment (luhmirin-s) [inline fingerprint/connect/src/main/java/com/simprints/fingerprint/connect/screens/controller/ConnectScannerControllerFragment.kt:74]: This is just a styling nitpick, but the old line was significantly better in terms of readability, IMO.

File: fingerprint/connect/src/main/java/com/simprints/fingerprint/connect/screens/controller/ConnectScannerControllerFragment.kt [hunk 3]
```diff
@@ -91,92 +99,135 @@ internal class ConnectScannerControllerFragment : Fragment(R.layout.fragment_con
...
-                    ScannerOffFragmentArgs(screen.currentScannerId).toBundle()
-                )
+        if (!fragmentViewModel.isInitialized) {
+            activityViewModel.init(args.params)
+            fragmentViewModel.isInitialized = true
+        }
 
-                is ConnectScannerIssueScreen.ScannerError -> screen.currentScannerId?.let { showKnownScannerDialog(it) }
+        findNavController().handleResult(
+            this,
```
Comment (luhmirin-s) [inline fingerprint/connect/src/main/java/com/simprints/fingerprint/connect/screens/controller/ConnectScannerControllerFragment.kt:111]: Might as well add the arg names if breaking them up into new lines.

File: fingerprint/connect/src/main/java/com/simprints/fingerprint/connect/screens/controller/ConnectScannerControllerFragment.kt [hunk 4]
```diff
@@ -91,92 +99,135 @@ internal class ConnectScannerControllerFragment : Fragment(R.layout.fragment_con
...
+
+                    is ConnectScannerIssueScreen.ScannerOff -> internalNavController?.navigateSafely(
+                        currentlyDisplayedInternalFragment,
+                        R.id.issueScannerOffFragment,
+                        ScannerOffFragmentArgs(screen.currentScannerId).toBundle()
+                    )
+
+                    is ConnectScannerIssueScreen.ScannerError -> screen.currentScannerId?.let {
+                        showKnownScannerDialog(
+                            it
```
Comment (luhmirin-s) [inline fingerprint/connect/src/main/java/com/simprints/fingerprint/connect/screens/controller/ConnectScannerControllerFragment.kt:162]: Auto-formatting does not ceases to amaze me :D

File: infra/config-store/src/test/java/com/simprints/infra/config/store/local/migrations/ProjectConfigSharedPrefsMigrationTest.kt [hunk 5]
```diff
@@ -9,10 +9,27 @@ import com.simprints.infra.authstore.AuthStore
 import com.simprints.infra.config.store.local.migrations.ProjectConfigSharedPrefsMigration.Companion.ALL_KEYS
 import com.simprints.infra.config.store.local.migrations.ProjectConfigSharedPrefsMigration.Companion.PROJECT_SETTINGS_JSON_STRING_KEY
 import com.simprints.infra.config.store.local.migrations.models.OldProjectConfig
-import com.simprints.infra.config.store.local.models.*
```
Comment (luhmirin-s) [inline infra/config-store/src/test/java/com/simprints/infra/config/store/local/migrations/ProjectConfigSharedPrefsMigrationTest.kt:12]: Why all those wildcard replacements? Feels unnecessary, especially in test imports.

File: infra/event-sync/src/main/java/com/simprints/infra/eventsync/event/remote/models/ApiAlertScreenPayload.kt [hunk 6]
```diff
@@ -5,7 +5,43 @@ import com.simprints.infra.config.store.models.TokenKeyType
 import com.simprints.infra.events.event.domain.models.AlertScreenEvent.AlertScreenPayload
 import com.simprints.infra.events.event.domain.models.AlertScreenEvent.AlertScreenPayload.AlertScreenEventType
 import com.simprints.infra.eventsync.event.remote.models.ApiAlertScreenPayload.ApiAlertScreenEventType
-import com.simprints.infra.eventsync.event.remote.models.ApiAlertScreenPayload.ApiAlertScreenEventType.*
```
Comment (luhmirin-s) [inline infra/event-sync/src/main/java/com/simprints/infra/eventsync/event/remote/models/ApiAlertScreenPayload.kt:8]: This is where wildcard hate crosses the line :D Please leave the wildcard.

File: infra/events/src/main/java/com/simprints/infra/events/event/local/EventRoomDatabase.kt [hunk 7]
```diff
@@ -7,10 +7,23 @@ import androidx.room.Room
...
+import com.simprints.infra.events.event.local.migrations.EventMigration3to4
+import com.simprints.infra.events.event.local.migrations.EventMigration4to5
+import com.simprints.infra.events.event.local.migrations.EventMigration5to6
+import com.simprints.infra.events.event.local.migrations.EventMigration7to8
+import com.simprints.infra.events.event.local.migrations.EventMigration8to9
+import com.simprints.infra.events.event.local.migrations.EventMigration9to10
 import com.simprints.infra.events.event.local.models.DbEvent
 import com.simprints.infra.events.event.local.models.DbEventScope
-import com.simprints.infra.events.local.migrations.*
+import com.simprints.infra.events.local.migrations.EventMigration6to7
```
Comment (luhmirin-s) [inline infra/events/src/main/java/com/simprints/infra/events/event/local/EventRoomDatabase.kt:26]: This seems out of place.

**Review by luhmirin-s — APPROVED** [review]
The contents are great, but I strongly disagree about some code-style choices.

File: feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorFragment.kt (hunk 1 above)
Comment (alexandr-simprints) [reply feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorFragment.kt:80]: Yes, that's right. The formatting was applied.

File: infra/config-store/src/test/java/com/simprints/infra/config/store/local/migrations/ProjectConfigSharedPrefsMigrationTest.kt (hunk 5 above)
Comment (alexandr-simprints) [reply infra/config-store/src/test/java/com/simprints/infra/config/store/local/migrations/ProjectConfigSharedPrefsMigrationTest.kt:12]: Auto-formatting imports on commit was turned on

File: infra/events/src/main/java/com/simprints/infra/events/event/local/EventRoomDatabase.kt (hunk 7 above)
Comment (alexandr-simprints) [reply infra/events/src/main/java/com/simprints/infra/events/event/local/EventRoomDatabase.kt:26]: Auto-formatting imports on commit was turned on

Comment (luhmirin-s) [reply infra/events/src/main/java/com/simprints/infra/events/event/local/EventRoomDatabase.kt:26]: I meant the import itself, it seems that the file has a wrong package declaration.

**Pull request merged**